### PR TITLE
Add more changelog possible cases

### PR DESCRIPTION
**Pull request by IanCa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,19 +90,25 @@ corrections in tag values or units.
...
-| Node moved in schema without change in meaning. | Minor |
-| Revision of description field in schema.        | Patch   |
-| Correction of suggestedTag or relatedTag.       | Patch  |
-| Correction of wiki syntax such as closing tags. | Patch |
+| Change                                              | Semantic-level |
+|-----------------------------------------------------|----------------|
+| Major addition to HED functionality                 | Major          |
+| Tag deleted from schema.                            | Major          |
+| Unit or unit class removed from node.               | Major          |
+| Node attribute value changed (excluding below)      | Major          |
```
Comment (VisLab) [inline README.md:98]: I don't think this should trigger a major version change

File: README.md [hunk 2]
```diff
@@ -90,19 +90,25 @@ corrections in tag values or units.
...
+| New unit class or unit added to schema.             | Minor          |
+| New unit class added to node.                       | Minor          |
+| New value class added to node.                      | Minor          |
+| Node moved in schema without change in meaning.     | Minor          |
+| Revision of description field in schema.            | Patch          |
+| Correction of suggestedTag or relatedTag.           | Patch          |
+
 
 **Note:** It is an official policy that once in a schema, a node will not be removed.
 If a node becomes out-of-date, a `deprecated` attribute will be added to the tag in the schema.
```
Comment (VisLab) [inline README.md:114]: Replace:
>If a node becomes out-of-date, a `deprecated` attribute will be added to the tag in the schema.

with:
> If a node becomes out-of-date during a minor version update, a `deprecated` attribute will be added to the tag in the schema.

File: README.md [hunk 3]
```diff
@@ -90,19 +90,25 @@ corrections in tag values or units.
...
-| Unit or unit class removed from node.   | Major  |
-| New tag added to the schema.            | Minor  |
-| New attribute added to schema.          | Minor  |
-| New unit class or unit added to schema. | Minor  |
-| New unit class added to node.           | Minor  |
-| Node moved in schema without change in meaning. | Minor |
-| Revision of description field in schema.        | Patch   |
-| Correction of suggestedTag or relatedTag.       | Patch  |
-| Correction of wiki syntax such as closing tags. | Patch |
+| Change                                              | Semantic-level |
```
Comment (VisLab) [inline README.md:93]: Add the following at the end of the previous paragraph:

Major version changes to the schema may be backwards incompatible and thus impact existing annotations. Guidance on how existing annotations should be updated will be provided with major schema version releases.

File: README.md (hunk 1 above)
Comment (IanCa) [reply README.md:98]: Node attribute values could include things like removing a valueClass, making a unit a prefix now, or other quirky behaviors.  

I just didn't think it was worth trying to explicitly cover all cases, given most are very unlikely.(e.g. removing a modifier could also have catastrophic effects without notification, but is all but assured to never happen)

I'm not very opinionated, just that was the thought behind why I marked in major.

Comment (VisLab) [reply README.md:98]: I'm going to merge and we can review again.

Comment (VisLab) [inline README.md:98]: I don't think that None attribute change should be Major.  Please change it to Minor.

**Pull request merged**